client/src/shared/components/Text-field/text-field.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
```

Comment:
- filename should be `index.tsx`
![image](https://user-images.githubusercontent.com/73863685/223047617-08b3ee18-3b33-4c56-907b-6cfdf61c38e9.png)